build.sbt

```diff
@@ -1,24 +1,27 @@
 name := "scala-iso"
-scalaVersion := "2.13.3"
-crossScalaVersions := Seq("2.12.13", "2.13.3")
+scalaVersion := "3.7.0"
 
-inThisBuild(List(
-  organization := "io.bartholomews",
-  homepage := Some(url("https://github.com/bartholomews/scala-iso")),
-  licenses += ("MIT", url("https://opensource.org/licenses/MIT")),
-  developers := List(
-    Developer(
-      "bartholomews",
-      "Federico Bartolomei",
-      "scala-iso@bartholomews.io",
-      url("https://bartholomews.io")
+inThisBuild(
+  List(
+    organization := "io.bartholomews",
+    homepage := Some(url("https://github.com/viperey/scala-iso")),
+    licenses += ("MIT", url("https://opensource.org/licenses/MIT")),
+    developers := List(
+      Developer(
+        "bartholomews",
+        "Federico Bartolomei",
+        "scala-iso@bartholomews.io",
+        url("https://bartholomews.io")
+      )
     )
   )
-))
+)
+
+scalacOptions += "-Yretain-trees"
```

Comment:
to make it work in cross version (i.e. `+compile`) this should work
```
scalacOptions ++= {
  CrossVersion.partialVersion(scalaVersion.value) match {
    case Some((3, _)) => Seq("-Yretain-trees")
    case Some((2, _)) => Seq.empty
    case _ => Seq.empty
  }
}
```

Reply:
:+1: